aws_lambda_builders/utils.py

```diff
@@ -29,17 +29,25 @@ def copytree(source, destination, ignore=None):
     :param ignore:
         A function that returns a set of file names to ignore, given a list of available file names. Similar to the
         ``ignore`` property of ``shutils.copytree`` method
+
+    :type include: function
```

Comment:
nit: Callable type instead of function for easy of read
```suggestion
    :type include: Callable[[str], bool]
```